### ISSUE TITLE
Dealing with Logit/Probit results 

### ISSUE THREAD

**Issue by horn1994** [mwburke/stargazer#24]
Hi all,

Is there a way to remove R-squared stats? Since those are not compatible with logit/probit models I'm getting an error. Also, is there a way to return marginal effects? Statsmodels has this feature but does not seem to work with stargazer.

**Comment by toobaz** [comment]
Fixed in #47 : while still not optimal (you have to manually hide the unused lines), you should now be able to represent logit/probit models.

**Comment by llinfeng** [comment]
Here is [pointer](https://github.com/mwburke/stargazer/pull/47#issuecomment-643835735) for "manually hiding the empty $R^2$ and adjusted-$R^2$ rows". Assuming the Stargazer object is assigned to `table`, the following one liner gets ride of the two empty lines:
```
table.show_adj_r2 = table.show_r2 = False
```

**Comment by llinfeng** [comment]
@toobaz For the marginal effects obtained using `get_margeff()` for Probit models, is there a straightforward path to pass it to `Stargazer([model1, model])`? I spent an hour trying to tweak the code, but failed to get anything working.

Alternatively, suppose I have a number of dataframes each with two columns, `"coef."` and `"p-value"`. Suppose also that each of these dataframe is indexed by covairate names. Is there a straightforward way to compose a "regression table" off these dataframes? I searched around and did not find a good Python implementation. For R, people suggested the `texreg` library.
